docs/dml-of-carbondata.md

```diff
@@ -37,13 +37,28 @@ CarbonData DML statements are documented here,which includes:
   This command is used to load csv files to carbondata, OPTIONS are not mandatory for data loading process. 
 
   ```
-  LOAD DATA INPATH 'folder_path'
+  LOAD DATA INPATH 'folder_path' [ OVERWRITE ] 
   INTO TABLE [db_name.]table_name 
   OPTIONS(property_name=property_value, ...)
   ```
   **NOTE**:
-    * Use 'file://' prefix to indicate local input files path, but it just supports local mode.
-    * If run on cluster mode, please upload all input files to distributed file system, for example 'hdfs://' for hdfs.
+   * Use 'file://' prefix to indicate local input files path, but it just supports local mode.
+
+   * If run on cluster mode, please upload all input files to distributed file system, for example 'hdfs://' for hdfs.
+
+* [ OVERWRITE ] :
+
+  ​	By default, new data is appended to the table. If `OVERWRITE` is used, the table is instead overwritten with new data.
+
+  ​	Example:   
+
+  ```sql
+  CREATE TABLE carbon_load_overwrite(id int, name string, city string, age int)
```

Comment:
i think, no need to add create table here. Just load command example should be enough